File: 251208/최소공배수 구하기/find-the-least-common-multiple.py

#1
'''
import math

a,b= map(int,input().split())
lcm_value = math.lcm(a,b)

print(lcm_value)

'''

#2

a,b= map(int,input().split())

def make_gcd(a,b):
    while b:
	    a,b=b,a%b
    return a

def make_lcm(a,b):
    gcd_val =make_gcd(a,b) 
    return (a*b//gcd_val)

print(make_lcm(a,b))

File: 251208/최소공배수 구하기/find-the-least-common-multiple.cpp
//C++

#include <iostream>
#include <bits/stdc++.h>

using namespace std;

int make_gcd(int a, int b){
    while(b!=0){
	int remain=a%b;
	a=b;
	b=remain;    
    }
    return a;
}

int make_lcm(int a, int b){
    int gcd_val=make_gcd(a,b);

    return (a*b)/gcd_val;
}

int main(){
    int a, b;
    cin>>a>>b;    
    cout<<make_lcm(a,b);

}
